explorer/src/Pos/Explorer/Web/Server.hs

```diff
@@ -366,6 +370,32 @@ getAddressSummary cAddr = do
             ATRedeem     -> CRedeemAddress
             ATUnknown {} -> CUnknownAddress
 
+
+getAddressUtxoBulk
+    :: (HasNodeConfiguration, ExplorerMode ctx m)
+    => [CAddress]
+    -> m [CUtxo]
+getAddressUtxoBulk cAddrs = do
+    when (not explorerExtendedApi) $
```

Comment:
If you insist this is the way to go then restrict the number of `CAddress` you want to be able to send to something sensible. Maybe 10 max?
```
when (length cAddrs) >= 10 $
        throwM err405
        { errReasonPhrase = "Maximum number of addresses you can send to fetch Utxo in bulk is 10!"
        }
```

Reply:
In this case it was more a matter of need for my use case (a light wallet with multiple addresses) given the current performance limitations of the database when fetching utxo for a single address, but I believe that it would be generally useful to have a bulk API for addresses. E.g. getting a paginated chronologically ordered transaction history for multiple addresses would be quite cumbersome without a bulk API, which is a different issue of course, but I had it in mind when adding this first bulk endpoint.

Ok, I restricted the amount of addresses to 10, as you suggested. Maybe it would be nice to have this constant customizable in the config but I don't want to pollute the config file too much and I see that other constants around are also hard coded. What do you think @ksaric ?

Reply:
As a first approximation, I always tend to set the constants in the code itself, and then, if necessary, export it to a configuration file.
That way, the configuration won't be "polluted" and anyone who understands how to read Haskell code can change it (and rebuild it)...